src/docs/swagger.js

```diff
@@ -5,172 +5,324 @@ dotenv.config();
 const { APP_BASE_URL } = process.env;
 
 export default {
-  swagger: '2.0',
+  swagger: "2.0",
   info: {
-    description: `Welcome the documentation of Authors Haven API. The base url for working with this api is ${APP_BASE_URL}/api`,
-    version: '1.0.0',
-    title: 'Authors Haven API',
+    description: `Welcome to Authors Haven's API documentation. The base url for the API is ${APP_BASE_URL}/api`,
+    version: "1.0.0",
+    title: "Authors Haven API",
     contact: {
-      name: 'Team Balder 42',
-      url: 'https://github.com/andela/balder-ah-backend'
+      name: "Team Balder 42",
+      url: "https://github.com/andela/balder-ah-backend"
     },
     license: {
-      name: 'MIT',
-      url: 'https://opensource.org/licenses/MIT'
+      name: "MIT",
+      url: "https://opensource.org/licenses/MIT"
     }
   },
   host: APP_BASE_URL,
-  basePath: '/api',
-  consumes: ['application/json'],
-  produces: ['application/json'],
+  basePath: "/api",
+  consumes: ["application/json"],
+  produces: ["application/json"],
   tags: [
     {
-      name: 'users',
+      name: "users",
```

Comment:
Good job documenting your API, albeit, I can see you're using double quotes instead of single quotes, is there any reason why?